extensions/azure/data-plane-azure-storage/src/main/java/org/eclipse/dataspaceconnector/azure/dataplane/azurestorage/validator/AzureStorageValidator.java

```diff
@@ -0,0 +1,124 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+package org.eclipse.dataspaceconnector.azure.dataplane.azurestorage.validator;
+
+
+import org.eclipse.dataspaceconnector.common.string.StringUtils;
+
+import java.util.Base64;
+import java.util.regex.Pattern;
+
+
+/**
+ * Validates storage account resource names and keys.
+ */
+public class AzureStorageValidator {

```

Comment:
Are these rules for Azure Storage names publicly available? How stable are they? Consider using validation utilities in the SDK to that we don't have to hard code these rules ourselves (if available)

Reply:
Consider adding a link to the docs